packages/ketcher-react/src/script/ui/views/modal/components/toolbox/Bond/Bond.module.less

```diff
@@ -31,9 +31,8 @@
     font-size: 12px;
     color: @color-text-secondary;
 
-    select,
-    input {
-      margin: 4px 0 0 0;
+    span {
+      margin-bottom: 4px
     }
```

Comment:
It may be worth connecting HTML elements together using "&" and ">" so that styles are not overridden, just in case

Reply:
fixed